orchestrator-salt/src/main/resources/salt/pillar/top.sls

```diff
@@ -24,6 +24,10 @@ base:
     - match: compound
     - sssd.ipa
 
+  'G@roles:ipa_leave':
+    - match: compound
+    - sssd.ipa
```

Comment:
it should be ipa-leave I think

Reply:
this is the pillar top file to use the admin credentials